html_css/css_flexbox.md

```diff
@@ -0,0 +1,27 @@
+### Introduction
+
+The Flexible Box Module, usually referred to as flexbox, was designed as a one-dimensional layout model, and as a method that 
```

Comment:
This is a bit highbrow, bearing in mind that most of our users are beginners. I hate to use this phrase but could we "dumb this" paragraph down please? 😆